components/camel-jira/src/main/java/org/apache/camel/component/jira/producer/AddIssueLinkProducer.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.jira.producer;
+
+import static org.apache.camel.component.jira.JiraConstants.CHILD_ISSUE_KEY;
```

Comment:
I think the static imports should be at the bottom of the imports. Did you run the build with sourcecheck enabled?

Reply:
My bad, I use IDEA and had my imports organised in a different order. I'll format this to be consistent with the rest of the project.